src/main/java/org/takes/facets/auth/social/PsFacebook.java

```diff
@@ -66,13 +78,51 @@
     private final transient String key;
 
     /**
+     * Request and Response for testing.
+     */
+    private final transient Map.Entry<com.jcabi.http.Request,
+        WebRequestor.Response> testing;
+
+    /**
      * Ctor.
      * @param fapp Facebook app
      * @param fkey Facebook key
      */
     public PsFacebook(final String fapp, final String fkey) {
+        this(fapp, fkey, null);
```

Comment:
see http://www.yegor256.com/2014/05/13/why-null-is-bad.html we don't use NULL, ever